src/global_scheduler/global_scheduler.c

```diff
@@ -53,13 +53,21 @@ void free_global_scheduler(global_scheduler_state *state) {
   db_disconnect(state->db);
   utarray_free(state->local_schedulers);
   destroy_global_scheduler_policy(state->policy_state);
-  /* delete the plasma 2 photon association map */
+  /* Delete the plasma 2 photon association map. */
   HASH_ITER(hh, state->plasma_photon_map, entry, tmp) {
     HASH_DELETE(hh, state->plasma_photon_map, entry);
     /* Now deallocate hash table entry. */
     free(entry->aux_address);
     free(entry);
   }
+  /* Free the scheduler object info table. */
+  scheduler_object_info *object_entry, *tmp_entry;
+  HASH_ITER(hh, state->scheduler_object_info_table, object_entry, tmp_entry) {
+    HASH_DELETE(hh, state->scheduler_object_info_table, object_entry);
+    utarray_free(object_entry->object_locations);
```

Comment:
Could be mistaken, but from this example code, it looks like we don't have to. http://www.cs.bu.edu/~jappavoo/Resources/psml/apps/hashy/uthash/doc/utarray.html#_strings